code/game/gamemodes/objective_items.dm

```diff
@@ -32,11 +32,11 @@
 	excludefromjob = list("Captain")
 
 /datum/objective_item/steal/hoslaser
-	name = "the head of security's personal laser gun."
+	name = "the head of security's personal weapon."
```

Comment:
Assuming max levels of modularity, this should be overriden in `/modular_tempt/code/game/gamemodes/objective_items.dm`
But we're a fork of Splurt so it probably doesn't matter.

Reply:
Its almost liket hey took it from https://github.com/ViceStation/ViceStation/pull/9 without reading anything.

Reply:
> @NullFag Yeah I am just going to say this... this entire PR was taken from another station that isn't a fork of SPLURT anymore. Here is a direct link to the [ViceStation/ViceStation#9](https://github.com/ViceStation/ViceStation/pull/9) which is why nothing is modular.

I'm aware that it is copied from Vice, and I hope you're aware that this is not an actual excuse for @CipherSnagger to be lazy.

Reply:
> > @NullFag Yeah I am just going to say this... this entire PR was taken from another station that isn't a fork of SPLURT anymore. Here is a direct link to the [ViceStation/ViceStation#9](https://github.com/ViceStation/ViceStation/pull/9) which is why nothing is modular.
> 
> I'm aware that it is copied from Vice, and I hope you're aware that this is not an actual excuse for @CipherSnagger to be lazy.

I mostly just want credit for my hard work. The fact he's lazy and didn't read a damn thing is on him.

Reply:
> I mostly just want credit for my hard work. The fact he's lazy and didn't read a damn thing is on him.

I don't have much to say except the fact I understand the feeling, seeing as SPLURT has been doing [this](https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/294) [exact](https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/277) thing to me.

Reply:
> > I mostly just want credit for my hard work. The fact he's lazy and didn't read a damn thing is on him.
> 
> I don't have much to say except the fact I understand the feeling, seeing as SPLURT has been doing [this](https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/294) [exact](https://github.com/SPLURT-Station/S.P.L.U.R.T-Station-13/pull/277) thing to me.

Yeah that's Mosley in a nutshell. Make sure to keep Cipher in check if he ports things from other servers. Goes for the rest of the Tempt team. Its getting on me and my teams nerves.